### PR TITLE
[Contact] covered Model Config by Unit Test

### PR DESCRIPTION
**Pull request by srsathish92**
### Description (*)
[Contact] covered Model Config by Unit Test
Path: `app/code/Magento/Contact/Model/Config.php`

### Manual testing scenarios (*)
`vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist app/code/Magento/Contact/Test/Unit/Model/ConfigTest.php`

### Questions or comments
If any please let me know your feedback

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Contact/Test/Unit/Model/ConfigTest.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    }
+
+    /**
+     * Test isEnabled()
+     *
+     * @return void
+     */
+    public function testIsEnabled(): void
+    {
+        $this->scopeConfigMock->expects($this->once())
```
Comment (dmytro-ch) [inline app/code/Magento/Contact/Test/Unit/Model/ConfigTest.php:61]: No need to duplicate the logic for `testIsEnabled` and `testIsNotEnabled`. Please use a data provider instead.

**Review by dmytro-ch — CHANGES_REQUESTED** [review]

File: app/code/Magento/Contact/Test/Unit/Model/ConfigTest.php (hunk 1 above)
Comment (srsathish92) [reply app/code/Magento/Contact/Test/Unit/Model/ConfigTest.php:61]: @dmytro-ch Thanks for the feedback, I have removed `testIsNotEnabled` and added data provider.

**Comment by dmytro-ch** [comment]
@srsathish92, thank you for the updates.

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6484](https://jira.corp.magento.com/browse/ENGCOM-6484) has been created to process this Pull Request
:eight_spoked_asterisk: @dmytro-ch, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Charlie** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**